analyses/molecular-subtyping-pathology/run-subtyping-aggregation.sh

```diff
@@ -26,4 +26,4 @@ Rscript -e "rmarkdown::render('02-incorporate-clinical-feedback.Rmd', clean = TR
 Rscript -e "rmarkdown::render('03-incorporate-pathology-feedback.Rmd', params=list(is_ci = ${IS_CI}), clean = TRUE)"
 
 # Run the meningioma clinical subtyping step
-Rscript -e "rmarkdown::render('clinical-subtyping-meningioma.Rmd', clean = TRUE)"
+#Rscript -e "rmarkdown::render('clinical-subtyping-meningioma.Rmd', clean = TRUE)"
```

Comment:
Ah yes, I forgot to mention this. This script needs `pbta-histologies.tsv` which is actually will be created later on in `molecular-subtyping-integrate` in the last step of the subtyping workflow in https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/scripts/run-for-subtyping.sh

My suggestion would be we can either make a logic to only run this out of the subtyping workflow or put it in as part of `molecular-subtyping-integrate` instead of `molecular-subtyping-pathology`.

Reply:
cc @jharenza to comment on how we could handle this issue above

Reply:
Ahh, I forgot about the `harmonized_diagnosis` column being created later. @jaclyn-taroni and I initially went back and forth about where to put these extra subtypings resulting from information within `pathology_free_text_diagnosis`, of which meningioma is one. We decided upon adding them to `molecular-subtyping-pathology` as the most appropriate place, since this information was derive from pathology reports. What if we added the `harmonized_diagnosis` column to the `molecular-subtyping-pathology` module instead and kept meningioma here?

Reply:
I made a different PR #1047 to work on this issue, if we want to add the meningioma results to `pbta-hisotlogies.tsv` we would have to merge that PR before this one.